pkg/k8sapi/helpers_test.go

```diff
@@ -27,21 +27,27 @@ func TestListClusterSPIFFEIDs(t *testing.T) {
 	}
 
 	t.Run("list fails", func(t *testing.T) {
-		client := FailList(k8stest.NewClientBuilder().Build())
+		builder, err := k8stest.NewClientBuilder()
+		assert.NoError(t, err)
```

Comment:
This should be `require.NoError`, since continuing to run the test will just panic (same for the rest that were added)